js-routes.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
   s.summary = %q{Brings Rails named routes to javascript}
 
   s.add_runtime_dependency(%q<railties>, [">= 3.2"])
-  s.add_runtime_dependency(%q<sprockets-rails>, ["< 3"])
+  s.add_runtime_dependency(%q<sprockets-rails>)
```

Comment:
I never believed in those limitations: my gems get broken on a random version update from a vendor - minor, major or patch. Especially js-routes - it went broken on minor version of rails several times. That's why I think those up level limitations never help.